lib/Backend/BackwardPass.cpp

```diff
@@ -3227,9 +3227,11 @@ BackwardPass::ProcessBlock(BasicBlock * block)
                     loop->symClusterList->MapSet<BVSparse<JitArenaAllocator>*>(symID, [](SymID a, BVSparse<JitArenaAllocator> *symbols) {
                         symbols->Set(a);
                     }, syms);
+                    SymTable* symTable = loop->GetFunc()->m_symTable;
                     FOREACH_BITSET_IN_SPARSEBV(curSymID, syms)
                     {
-                        if (!loop->GetFunc()->m_symTable->Find(curSymID)->IsStackSym())
+                        Sym* potentialSym = symTable->Find(curSymID);
+                        if (potentialSym == nullptr || !potentialSym->IsStackSym())
                         {
                             syms->Clear(curSymID);
```

Comment:
Can you add a test for this ? When is a sym not found in the symTable ? Are there StackSyms not in symTable ?

Reply:
It looks like the underlying issue that allowed the nullptr entries into the symtable is now resolved; however, I'd like to keep this set up defensively like this until that's better understood.

Reply:
Do you want to add an assert in place ?  Or are there cases where it's legal for stack sym to not be in the function symtable ?

Reply:
Sure, Rest of the change LGTM.